docs/.vitepress/config.ts

```diff
@@ -43,10 +43,10 @@ export default withPwa(defineConfig({
     ['meta', { property: 'og:description', content: vitestDescription }],
     ['meta', { property: 'og:url', content: ogUrl }],
     ['meta', { property: 'og:image', content: ogImage }],
-    ['meta', { name: 'twitter:title', content: vitestName }],
-    ['meta', { name: 'twitter:description', content: vitestDescription }],
-    ['meta', { name: 'twitter:image', content: ogImage }],
-    ['meta', { name: 'twitter:card', content: 'summary_large_image' }],
+    ['meta', { name: 'x:title', content: vitestName }],
+    ['meta', { name: 'x:description', content: vitestDescription }],
+    ['meta', { name: 'x:image', content: ogImage }],
+    ['meta', { name: 'x:card', content: 'summary_large_image' }],
```

Comment:
Is that also expected? I still don't see meta tags updated on Twitter's own docs: 

https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started

Reply:
true, thank for clearing that out.
I updated it.